### PR TITLE
Upload qemu binaries to github release

### PR DESCRIPTION
**Pull request by nghiant2710**

### CODE REVIEW

**Comment by petrosagg** [comment]
I think we should set the QEMU version and hashes in the code, at least in the form of default values

**Comment by nghiant2710** [comment]
@petrosagg since we have version control with build config on jenkins, my initial idea is we keep QEMU version and commit hashes there so we don't need to PR this repo when we want to build new version. However we don't rebuild this regularly so I'm neutral here. Need to go back to bed, will update the PR later

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,8 @@
 FROM debian:jessie
 
+ENV QEMU_VERSION 2.5.0-resin
+ENV QEMU_SHA256 6525da6fef4beb5b60e2526f15fc2aa734e4395440bdd23b0ef189c57bbcea4f
```
Comment (nghiant2710) [inline Dockerfile]: @petrosagg , updated!

**Comment by petrosagg** [comment]
SGTM

**Pull request merged**